molo/core/content_import/helper.py

```diff
@@ -135,15 +140,29 @@ def import_categories_for_child_language(
             print "no source found for: ", category.source, (
                 SectionPage.objects.all().values('uuid'))
 
+    @classmethod
+    def get_main_language(cls, locales):
+        mains = [locale for locale in locales if locale.get('is_main')]
+
+        if not mains:
+            raise ImportError("No main languages have been given")
+        elif len(mains) > 1:
+            raise ImportError("Cannot have multiple main languages")
+        else:
+            return mains[0]
+
     def import_content_for(self, locales):
+        main = self.get_main_language(locales)
+        locales = [main] + [
+            locale for locale in locales if not locale.get('is_main')]
         for selected_locale in locales:
-
             site_language, _ = SiteLanguage.objects.get_or_create(
                 locale=Locale.parse(selected_locale.get('locale')).language,
                 is_main_language=selected_locale.get('is_main'))
 
             self.import_all_categories(site_language, selected_locale)
             self.import_all_pages(site_language, selected_locale)
+
         self.update_pages_with_linked_page_field()
```

Comment:
Can we add some tests for this method:
  - check that an `ImportError` is raised if `locales` has more than one language with `is_main == True`
  - check than an `ImportError` is raised if none of the locales have `is_main == True`
  - check that the main language no longer needs to be first in the list of `locales`